cmd/gcpPublishEvent.go

```diff
@@ -101,7 +101,23 @@ func runGcpPublishEvent(utils gcpPublishEventUtils) error {
 		return errors.Wrap(err, "failed to publish event")
 	}
 
-	log.Entry().Info("event published successfully!")
+	log.Entry().Infof("Event published successfully! gcpProjectNumber: %s, topic: %s", config.GcpProjectNumber, config.Topic)
```

Comment:
i think we can leave the project number since its an internal project and not exposed to end user, only the topic name should be enough for us to identify if the event is published to the correct topic

Reply:
Alright, I updated the log message.